client/i18n/locales/english/translations.json

```
@@ -849,7 +849,9 @@
     "opens-new-window": "Opens in new window",
     "rsa-checkbox": "I have tried the Read-Search-Ask method",
     "similar-questions-checkbox": "I have searched for similar questions that have already been answered on the forum",
-    "edit-my-profile": "Edit my profile"
+    "edit-my-profile": "Edit my profile",
+    "editor-a11y-off": "{{editorName}} editor content. Press Alt+F1 for accessibility options.",
```

```suggestion
    "editor-a11y-off": "{{editorName}} editor content. Press Alt+F1 for accessibility options.",
```

So on my mac, it's Option+F1 - We could just change this to `Alt/Option+F1` or `Alt+F1 or Option+F1` for now - or we could [do something like we do here](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/templates/Challenges/classic/lower-jaw.tsx#L301) - e.g. detect if it's mac and different i18n key/value.